learningequality/studio
# Better error handling in change event handlers

## Description

* Adds discrete error handling for non-bulk actions to prevent changes being reported as errored when only one in the batch has
* This was making #2621 more apparent, as errors during the creation of a single content node in a batch would then cause the frontend to prompt reinsertion of all the contentnodes
* Changes from using auto_now_add to using a default, to ensure that null values are not set when Django does an auto upsert
* Adds modified timestamps during bulk content node updates, adds tests for the same.
* Catches when tag relationship creation fails due to it already existing and ensures that the operation completes successfully.

#### Issue Addressed (if applicable)

* Fixes #2621
* Fixes #2546

## Steps to Test

- Copy a lot of items to the clipboard at once
- Make sure no errors appear
- Update content nodes
- Check that the 'last modified' for the channel changes
